editor/inspector_dock.cpp

```diff
@@ -541,9 +541,9 @@ void InspectorDock::update(Object *p_object) {
 	p->add_shortcut(ED_SHORTCUT("property_editor/expand_revertable", TTR("Expand Non-Default")), EXPAND_REVERTABLE);
 
 	p->add_separator(TTR("Property Name Style"));
-	p->add_radio_check_item(TTR("Raw"), PROPERTY_NAME_STYLE_RAW);
-	p->add_radio_check_item(TTR("Capitalized"), PROPERTY_NAME_STYLE_CAPITALIZED);
-	p->add_radio_check_item(TTR("Localized"), PROPERTY_NAME_STYLE_LOCALIZED);
+	p->add_radio_check_item(TTR("Raw") + " (like_this)", PROPERTY_NAME_STYLE_RAW);
+	p->add_radio_check_item(TTR("Capitalized") + " (Like This)", PROPERTY_NAME_STYLE_CAPITALIZED);
+	p->add_radio_check_item(TTR("Localized (Like This)"), PROPERTY_NAME_STYLE_LOCALIZED);
```

Comment:
This probably warrants a translator comment for context on how to translate "Like This".